src/main/java/de/zalando/ep/zalenium/container/DockerContainerClient.java

```diff
@@ -272,12 +306,20 @@ public ContainerCreationStatus createContainer(String zaleniumContainerName, Str
         extraHosts.addAll(hubExtraHosts);
 
         HostConfig.Builder hostConfigBuilder = HostConfig.builder()
-            .appendBinds(binds)
-            .networkMode(networkMode)
-            .extraHosts(extraHosts)
-            .autoRemove(true)
-            .storageOpt(storageOpt)
-            .privileged(isZaleniumPrivileged);
+                .appendBinds(binds)
+                .networkMode(networkMode)
+                .extraHosts(extraHosts)
+                .autoRemove(true)
+                .storageOpt(storageOpt)
+                .privileged(isZaleniumPrivileged);
+
+        String cpuLimit = getSeleniumContainerCpuLimit();
+        String memoryLimit = getSeleniumContainerMemoryLimit();
+
+        if (cpuLimit != null)
```

Comment:
If the env var is not set, does it really come back as null? Or does it come back as an empty string?

Perhaps you can use `Strings.isNullOrEmpty` from google.commons to cover both cases.